### PR TITLE
zsh-navigation-tools 2.1.11

### PR DESCRIPTION
**Pull request by psprint**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Version update, and also shorter `caveats`, as no `$fpath` update is needed anymore.

### CODE REVIEW

File: Formula/zsh-navigation-tools.rb [hunk 1]
```diff
@@ -17,7 +17,6 @@ def install
 
   def caveats; <<-EOS.undent
     To run zsh-navigation-tools, add the following at the end of your .zshrc:
-      fpath+=( #{HOMEBREW_PREFIX}/share/zsh-navigation-tools )
```
Comment (mikemcquaid) [inline Formula/zsh-navigation-tools.rb]: No long necessary

Comment (mikemcquaid) [inline Formula/zsh-navigation-tools.rb]: ?

Comment (psprint) [inline Formula/zsh-navigation-tools.rb]: Yes, I'm author of the software and from that version I've embedded fpath update within the sourced file

Comment (mikemcquaid) [inline Formula/zsh-navigation-tools.rb]: 👏 👍

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by psprint** [comment]
Thanks!

**Pull request closed without merging**